core/src/main/scala/kafka/admin/TopicCommand.scala

```diff
@@ -119,7 +119,7 @@ object TopicCommand extends Logging {
     }
 
     def hasUnderReplicatedPartitions: Boolean = {
-      info.isr.size < info.replicas.size
+      info.replicas.size - info.isr.size > 0
```

Comment:
Just double checking, but are we planning to update this logic in a separate patch?

Reply:
Yes, will upload it soon.